packages/strapi-parts/src/DatePicker/DatePicker.stories.mdx

```diff
@@ -12,7 +13,21 @@ This is the doc of the `DatePicker` component
 
 ## DatePicker
 
-Description...
+Date pickers help users selecting a specific date from a calendar. They can be associated to a Time picker component.  
+
+**Best practices**
+
+- When the date picker is clicked, it takes today's date.
```

Comment:
Not sure to understand that first point. What do you mean by "it takes" ?

Reply:
Is it also the case when another value has already been selected?

Reply:
- I probably should have said something like "it selects" or "it shows" or "it displays"? Does it sound better to you?
- Indeed, it's not the case. Once a date has been selected, this one is displayed and not _today's_ date.

Reply:
WYT of saying "activated" instead of "clicked"?

Reply:
Sounds good to me!